docker/base-deps/Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN apt-get update -y \
     && apt-get install -y sudo tzdata \
     && useradd -ms /bin/bash -d /home/ray ray --uid $RAY_UID --gid $RAY_GID \
     && usermod -aG sudo ray \
+    && chmod 777 /home/ray \
```

Comment:
Can you add a comment in the Dockerfile that explains why `chmod` is here?

Reply:
Updated!